### PR TITLE
Default to release when uploading maps

### PR DESCRIPTION
**Pull request by dsimmons87**
Default to using the latest "release" version when uploading maps, as that's the general use case except when explicitly testing a playtest. This is because there is no upgrade path from playtests.

Also adds a notice warning users when uploading a map to playtest

### CODE REVIEW

File: openra/static/style003.css [hunk 1]
```diff
@@ -1484,6 +1484,17 @@ p.registerInfo {
...
 
+.upload-map-playtest-warning {
+    margin-bottom:10px;
+    color:#FF0000;
+    background:white;
+    background:#FFEEEE;
+    margin-right:10px;
+    padding:10px;
+    border:1px solid #ff3333;
+    display:none;
```
Comment (abcdefg30) [inline openra/static/style003.css:1495]:
```suggestion
    margin-bottom: 10px;
    color: #FF0000;
    background: white;
    background: #FFEEEE;
    margin-right: 10px;
    padding: 10px;
    border: 1px solid #ff3333;
    display: none;

```

File: openra/templates/uploadMap.html [hunk 2]
```diff
@@ -9,9 +9,20 @@
 				$(".license_options").hide();
 			});
 
-			$('.upload-map-form-parser option:first-child').attr("selected", "selected");
+			function is_release_version(version) {
+				return /^release-/.test(version);
+			}
+
+			$('.upload-map-form-parser option').filter(function(){
```
Comment (abcdefg30) [inline openra/templates/uploadMap.html:16]:
```suggestion
			$('.upload-map-form-parser option').filter(function() {
```

File: openra/templates/uploadMap.html [hunk 3]
```diff
@@ -9,9 +9,20 @@
...
+				return is_release_version(this.value)
+			}).first().attr("selected", "selected");
 
 			$('.upload-map-form-parser option:first-child').append(' (latest version)')
+
+			$('.upload-map-form-parser select').on('click change', function(e){
+
+				$('.upload-map-playtest-warning').toggle(!is_release_version(e.target.value))
```
Comment (abcdefg30) [inline openra/templates/uploadMap.html:24]:
```suggestion
			$('.upload-map-form-parser select').on('click change', function(e) {
				$('.upload-map-playtest-warning').toggle(!is_release_version(e.target.value))
```

**Review by abcdefg30 — COMMENTED** [review]
Some minor style nits. Lgtm otherwise.

**Review by abcdefg30 — APPROVED** [review]
Ideally commits are squashed before merging.

**Comment by dsimmons87** [comment]
> 

It's apparently not enabled on this repo

**Pull request merged**